src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -31,9 +35,16 @@ public InterceptorConfig(LoggingInterceptor loggingInterceptor, CustomUserAuthen
     @Override
     public void addInterceptors(@NonNull InterceptorRegistry registry) {
         registry.addInterceptor(loggingInterceptor);
+        registry.addInterceptor(transactionInterceptor())
+                .addPathPatterns(FILINGS, TRANSACTIONS);
         registry.addInterceptor(new TokenPermissionsInterceptor())
                 .addPathPatterns(PARTNERSHIP);
         registry.addInterceptor(customUserAuthenticationInterceptor)
                 .addPathPatterns(PARTNERSHIP);
     }
+
+    @Bean
+    public TransactionInterceptor transactionInterceptor() {
+        return new TransactionInterceptor();
```

Comment:
Could invoke the constructor that adds a logging name space.
e.g. https://github.com/companieshouse/psc-filing-api/blob/b50e10855c783a7c3de5fdcf398379cd51b4775b/src/main/java/uk/gov/companieshouse/pscfiling/api/config/InterceptorConfig.java#L125

Reply:
Added the argument